modules/coverage-report/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ~ Licensed to the Apache Software Foundation (ASF) under one ~ or more
```

Comment:
wrong license header

Reply:
Same license in all the pom file in synapse

Reply:
I thought since we adding new module we have to use wso2 license header. If having the same License header in pom files is the requirement, then, this looks good. 👍